src/servlets/metaTags/index.ts

```diff
@@ -22,6 +22,7 @@ import {
 import { Context, MetaTagFormat, Playable } from './types'
 
 const CAN_EMBED_USER_AGENT_REGEX = /(twitter|discord)/
+const RELEASE_DATE_FORMAT = 'ddd MMM DD YYYY HH:mm:ss GMTZZ'
```

Comment:
@raymondjacobson @Kyle-Shanks does this look correct? Keeping `new Date(..)` makes the type checking complain (expects a string) and not doing anything makes momentjs warn that parsing fell back to `Date()` which is "unreliable"

Reply:
https://github.com/AudiusProject/audius-protocol/blob/4aa83d8d659082e959ba3b7e4061bbea13f39e18/discovery-provider/src/queries/get_tracks.py#L95

for what I referenced. Tested and seemed to work but just double checking - not sure I need the GMT part.